openrecords.py

```diff
@@ -0,0 +1,303 @@
+    """
+    from flask import url_for
+    from urllib.parse import unquote
+
+    output = []
+    for rule in app.url_map.iter_rules():
+        options = {}
+        for arg in rule.arguments:
+            if arg == "year":
+                from datetime import datetime
+
+                options[arg] = "{}".format(datetime.now().year)
+                continue
+            options[arg] = "[{}]".format(arg)
+
+        methods = ",".join(rule.methods)
+        url = url_for(rule.endpoint, **options)
+        from datetime import datetime
+
+        if str(datetime.now().year) in url:
+            url = url.replace(str(datetime.now().year), "[year]")
+        line = unquote("{:50} {:20} {}".format(rule.endpoint, methods, url))
+
+        output.append(line)
+
+    for line in sorted(output):
+        print(line)
+
+
+@app.cli.command()
+@click.option("--test-name", help="Specify tests (file, class, or specific test)")
+@click.option(
+    "--coverage/--no-coverage", default=False, help="Run tests under code coverage."
+)
+@click.option("--verbose", is_flag=True, default=False, help="Py.Test verbose mode")
+def test(test_name: str = None, coverage: bool = False, verbose: bool = False):
+    """Run the unit tests."""
+
+    if coverage and not os.environ.get("FLASK_COVERAGE"):
+        os.environ["FLASK_COVERAGE"] = "1"
+        os.execvp(sys.executable, [sys.executable] + sys.argv)
+
+    command = []
+
+    if verbose:
+        command.append("-v")
+
+    if test_name:
+        command.append("tests/{test_name}".format(test_name=test_name))
+    else:
+        command.append("tests/")
+
+    pytest.main(command)
+
+    if COV:
+        COV.stop()
+        COV.save()
+        print("Coverage Summary:")
+        COV.report()
+        COV.html_report()
+        COV.xml_report()
+
+
+@app.cli.command()
+@click.option(
+    "--length",
+    default=25,
+    help="Number of functions to include in the profiler report.",
+)
+@click.option(
+    "--profile-dir", default=None, help="Directory where profiler data files are saved."
+)
+def profile(length, profile_dir):
+    """
+    Start the application under the code profiler.
+    """
+    from werkzeug.contrib.profiler import ProfilerMiddleware
+
+    app.wsgi_app = ProfilerMiddleware(
+        app.wsgi_app, restrictions=[length], profile_dir=profile_dir
+    )
+    app.run()
+
+
+@app.cli.command()
+def deploy():
+    """
+    Run deployment tasks.
+    """
+    from flask_migrate import upgrade
+
+    # migrate database to latest revision
+    upgrade()
+
+    # pre-populate
+    list(
+        map(
+            lambda x: x.populate(),
+            (
+                Roles,
+                Agencies,
+                Reasons,
+                Users,
+                LetterTemplates,
+                EnvelopeTemplates,
+                CustomRequestForms,
+            ),
+        )
+    )
+
+    es_recreate()
+
+
+if __name__ == "__main__":
+    main(as_module=False)
```

Comment:
I think you can get rid of `as_module=False` as it's False by default.

Reply:
Updated.